chanbackup/multi.go

```diff
@@ -87,9 +88,13 @@ func (m Multi) PackToWriter(w io.Writer, keyRing keychain.KeyRing) error {
 		}
 	}
 
+	e, err := lnencrypt.KeyRingEncrypter(keyRing)
+	if err != nil {
+		return fmt.Errorf("unable to generate encrypt key %v", err)
+	}
```

Comment:
nit: add newline before comment. Or move the new code to between the comment and the `return` statement.